crates/bevy_pbr/src/deferred/mod.rs

```diff
@@ -10,7 +10,7 @@ use crate::{
     ViewLightsUniformOffset,
 };
 use bevy_app::prelude::*;
-use bevy_asset::{load_internal_asset, weak_handle, Handle};
+use bevy_asset::{embedded_asset, load_embedded_asset, Handle};
```

Comment:
`load_embedded_asset` looks unused?

Reply:
its used at the bottom of the file, line 412